### ISSUE TITLE
maxResults 50?

### ISSUE THREAD

**Issue by peltzel** [optilude/jira-cycle-extract#4]
When I run this, it returns only 50 results each time.  Is this a setting on my JIRA that is preventing my expected behavior?

**Comment by peltzel** [comment]
I updated query.py to pagenate through all the responses so you can handle more issues.  If you want, I can open a PR with the change. Just let me know.

**Comment by oneross** [comment]
@peltzel yes please on the PR!  I just ran across this problem.  Even if maintainers don't merge, I'd love to see how you did it.

**Comment by spascoe** [comment]
It is actually quite easy to get more results, by applying the -n=### command line parameter.  On the other hand, you then need to know how may issues to expect, I like the idea of iterating through results and getting all of the data.

**Comment by peltzel** [comment]
LMK if you need any help on it.  I've added some simple functionality to put the data in a MySQL DB, which is really useful for me, but may not be meaningful to anyone else.
